apisix/plugins/file-logger.lua

```diff
@@ -14,23 +14,23 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
-local log_util     =   require("apisix.utils.log-util")
-local core         =   require("apisix.core")
-local ngx          =   ngx
-local io_open      =   io.open
+local log_util = require("apisix.utils.log-util")
+local core = require("apisix.core")
+local ngx = ngx
+local io_open = io.open
+local io_output = io.output
 local is_apisix_or, process = pcall(require, "resty.apisix.process")
 
-
```

Comment:
Please don't remove the blank line

Reply:
Sorry, this was auto-formatted by the IDE.